src/kibana/plugins/visualize/editor/sidebar.html

```diff
@@ -1,7 +1,8 @@
 <div class="sidebar-container">
   <form class="sidebar-list"
         ng-submit="visualizeEditor.$invalid ? dontApply() : stageEditableVis()"
-        name="visualizeEditor">
+        name="visualizeEditor"
+        novalidate><!-- see http://goo.gl/9kgz5w -->
```

Comment:
To quote [this wufoo article](http://www.wufoo.com/html5/attributes/11-novalidate.html?num2=10), `novalidate` is used "if you plan to do your own client-side validation [...]". Since this is exactly the case, this seems semantically fine to me.